static/app/views/settings/account/apiTokenRow.tsx

```diff
@@ -20,15 +20,10 @@ function ApiTokenRow({token, onRemove, tokenPrefix = ''}: Props) {
   return (
     <StyledPanelItem>
       <Controls>
-        <TokenPreview aria-label={t('Token preview')}>
-          {tokenPreview(
-            getDynamicText({
-              value: token.tokenLastCharacters,
-              fixed: 'ABCD',
-            }),
-            tokenPrefix
-          )}
-        </TokenPreview>
+        {getDynamicText({
+          value: token.name ? token.name : '<no name>',
+          fixed: 'TokenName_1234',
+        })}
```

Comment:
I think what you could do here is just not render this component/function if `token.name` does not exist, and then not have to worry about doing the `'<no name>'`

Reply:
Might lead to a better UI experience too!

Reply:
Because 100% of tokens and people before this feature are going to have <no_name> since they never had one haha

Reply:
you don't really need ot use getDynamicText anymore since it was screenshot testing related i think

Reply:
I've removed the `<no name>` placeholder and use of `getDynamicText`. 👍